reviewer-notes/WEEK_7.md

```diff
@@ -0,0 +1,17 @@
+# Summary
+
+[Lesson page](https://learn.codethedream.org/ctd-node-express-class-lesson-9-using-query-parameters/)
+
+From the lesson page:
+
+> This is a difficult lesson, so take your time with it...
+
+This week, students finish the Store API assignment (04).
+
+Note that students' PRs will probably have the contents of this week and the
+previous week combined.
```

Comment:
To clarify, this is the first/only week that they work in the `04-store-api` directory right? Or is there some work there from week 6?